Sources/RemindersLibrary/CLI.swift

```diff
@@ -42,12 +42,18 @@ private struct Add: ParsableCommand {
         name: .shortAndLong,
         help: "The date the reminder is due")
     var dueDate: DateComponents?
+    
+    @Option(
+        name: .shortAndLong,
+        help: "The priority of the reminder  [0: None, 4: High, 5: Medium, 6: Low]")
+    var priority: Int
```

Comment:
Recommend defining a enum for this.

Reply:
Thanks for the comment. I have updated the code.